src/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs

```diff
@@ -28,48 +28,86 @@
         /// </summary>
         public string Instance { get; set; }
 
         /// <summary>
-        /// Mutually exclusive with Instance, allows users to use the enum instead of the explicit url.
+        /// Specific instance in the case of Azure Active Directory. 
+        /// This property is mutually exclusive with <see cref="Instance"/>. 
+        /// It allows users to use the enum instead of the explicit url.
         /// </summary>
         public AzureCloudInstance AzureCloudInstance { get; set; } = AzureCloudInstance.None;
 
         /// <summary>
+        /// The redirect URI (also known as Reply URI or Reply URL), is the URI at which Azure AD will contact back the application with the tokens.
+        /// This redirect URI needs to be registered in the app registration (https://aka.ms/msal-net-register-app).
+        /// In MSAL.NET, <see cref="T:IPublicClientApplication"/> define the following default RedirectUri values:
+        /// <list type="bullet">
+        /// <item><description><c>urn:ietf:wg:oauth:2.0:oob</c> for desktop (.NET Framework and .NET Core) applications</description></item>
+        /// <item><description><c>msal{ClientId}</c> for Xamarin iOS and Xamarin Android without broker (as this will be used by the system web browser by default on these
+        /// platforms to call back the application)
+        /// </description></item>
+        /// </list>
+        /// These default URIs could change in the future.
+        /// In <see cref="Microsoft.Identity.Client.IConfidentialClientApplication"/>, this can be the URL of the Web application / Web API.
+        /// </summary>
+        /// <remarks>This is especially important when you deploy an application that you have initially tested locally;
+        /// you then need to add the reply URL of the deployed application in the application registration portal</remarks>
+
         /// </summary>
         public string RedirectUri { get; set; }
 
         /// <summary>
+        /// Enables you to configure the level of logging you want. The default value is <see cref="LogLevel.Info"/>. Setting it to <see cref="LogLevel.Error"/> will only get errors
+        /// Setting it to <see cref="LogLevel.Warning"/> will get errors and warning, etc..
+        /// See https://aka.ms/msal-net-logging
         /// </summary>
         public LogLevel LogLevel { get; set; }
 
         /// <summary>
+        /// Flag to enable/disable logging of Personally Identifiable data (PII) data. 
+        /// PII logs are never written to default outputs like Console, Logcat or NSLog
+        /// Default is set to <c>false</c>, which ensures that your application is compliant with GDPR. You can set
+        /// it to <c>true</c> for advanced debugging requiring PII. See https://aka.ms/msal-net-logging
         /// </summary>
+        /// <seealso cref="IsDefaultPlatformLoggingEnabled"/>
         public bool EnablePiiLogging { get; set; }
```

Comment:
There seem to be 4 settings related to logging. Would we want to group them outside of this confgi objecT?